app/assets/stylesheets/all/main.css.scss

```diff
@@ -19,20 +19,37 @@ a { color: $orange; }
 h1, h2, h3 { color: black; font-family: 'Open Sans', Helvetica, sans-serif; }
 h1 { margin: 0 0 10px 0; font-size: 17pt; }
 
-h1.title {
-  color: #777;
-  font-weight: bold;
-  margin-bottom: 28px;
-  i {
-    color: $light-gray;
-    margin-right: 5px;
+div#title_container {
```

Comment:
There are several places other than the main layout where `<h1 class="title">` occurs. Since they aren't wrapped in `#title_container`, they have now lost their styling. I'd suggest moving `h1.title` out of this block so that it still works even if there is no container.